cli/src/lib/deploy.ts

```diff
@@ -338,6 +345,20 @@ export async function deployImpl(this: Deploy, cl: typeof Deploy) {
   await disconnectFluenceClient();
 }
 
+async function getDealStatus(dealId: string) {
+  const { DealStatus } = await import("@fluencelabs/deal-ts-clients");
+  const { readonlyDealClient } = await getReadonlyDealClient();
+  const deal = readonlyDealClient.getDeal(dealId);
+  const status = Number(await deal.getStatus());
+
+  function isDealStatus(status: number): status is DealStatus {
```

Comment:
Maybe we can move that function outside `getDealStatus`. What do you think about it?

Reply:
I put it inside cause this function is typescript-pleaser utility one-liner. No real need to use it or have it globally in the module. This should always return true except in the case if deal-ts-clients is not updated properly. If you don't mind I will leave it as it is cause it works totally fine either way in my opinion